my-app/src/App.tsx

```diff
@@ -1,25 +1,52 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { useState } from 'react'
+import { AddTodoForm } from './AddTodoForm';
+import { TodoList } from './TodoList';
+
+const initialTodos: Todo[] = [
+  {
+    text: 'Walk the dog',
+    complete: false,
+    id: 'test1' + new Date().getTime(),
+  },
+  {
+    text: 'Write app',
+    complete: true,
+    id: 'test2' + new Date().getTime(),
+  },
+];
+
+function App() {  
+  const [todos, setTodos] = useState(initialTodos);
+
+  const toggleTodo: ToggleTodo = (selectedTodo: Todo) => {
+    const newTodos = todos.map(todo => {
+      if (todo === selectedTodo) {
+        return {
+          ...todo,
+          complete: !todo.complete,
+        };
+      }
+      return todo;
```

Comment:
Here, I would recommend the uses of a ternary
```suggestion
        return {
          ...todo,
          complete: todo === selectedTodo ? !todo.complete : todo.complete
        };
```